lib/aggregation_cursor.js

```diff
@@ -130,6 +130,11 @@ inherits(AggregationCursor, Readable);
 for (var name in CoreCursor.prototype) {
   AggregationCursor.prototype[name] = CoreCursor.prototype[name];
 }
+if (Symbol.asyncIterator) {
+  AggregationCursor.prototype[
+    Symbol.asyncIterator
```

Comment:
formatting here is not consistent with lib/cursor.js and lib/command_cursor.js
Is the newline necessary?

Reply:
It's a prettier rule that restricts the maximum length of a single line.